encoding/x/protobuf/inbound.go

```diff
@@ -78,9 +81,16 @@ func (u *unaryHandler) Handle(ctx context.Context, transportRequest *transport.R
 		}
 		responseData = protoBuffer.Bytes()
 	}
+	if isRawResponse(transportRequest.Headers) {
```

Comment:
The only way to accomplish this is to do this on the headers, believe me I tried other methods, unsuccessfully. I think we can start discussing this on https://github.com/yarpc/yarpc-go/issues/905 but we may want another issue.